src/Workspaces/Core/Portable/Options/OptionService.cs

```diff
@@ -4,197 +4,236 @@
 using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.Composition;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Host.Mef;
 using Microsoft.CodeAnalysis.Options.Providers;
 using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.Options
 {
-    [Export(typeof(IOptionService))]
```

Comment:
Just checked with @paulvanbrenk   It doesn't seem so.  They do read/write options.  But they go through Workspace.Options and never through IOptionService itself.